### PR TITLE
Lesson 1.2

### PR DESCRIPTION
**Pull request by secch97**
- Moved list into new `TodoList` component. Changed ReactDOM.render to ReactDOM.createRoot().
- Created `AddTodo` Form Component.
- Added components folder. Moved `AddTodoForm `and `TodoList `components to `components` folder. Changed components extension to .jsx.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,10 +1,9 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import ReactDOM from 'react-dom/client';
 import App from './App';
 
-ReactDOM.render(
+ReactDOM.createRoot(document.getElementById('root')).render(
```
Comment (viniciuspietscher) [inline src/index.js:5]: Great job figuring out how to get rid of the warning in the console! :1st_place_medal:

File: src/components/TodoList.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            );
+          })
+        }
+    </ul>
+  );
+};
+
+export {
+    TodoList as default
+};
```
Comment (viniciuspietscher) [inline src/components/TodoList.jsx:46]: :100:

File: src/App.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    /* Fragment creation */
+    <>
+      <h1>Todo List</h1>
+      <AddTodoForm/>
+      <TodoList/>
+    </>
+  );
+};
+
+export default App;
```
Comment (viniciuspietscher) [inline src/App.jsx:16]: Awesome! :100:

File: public/index.html [hunk 4]
```diff
@@ -1,33 +1,31 @@
...
+      ==================================================
+    -->
+    <link rel="icon" href="%PUBLIC_URL%/icons/react-todo-icon.ico" />
+    <!--
+      ==================================================
+      =                      TITLE                     =
+      ==================================================
     -->
-    <title>React App</title>
+    <title>To Do App</title>
```
Comment (viniciuspietscher) [inline public/index.html:25]: I don't think I changed my title when I did this course haha. Great job! :100:

**Review by viniciuspietscher — COMMENTED** [review]
:100: Approved!

File: src/index.js (hunk 1 above)
Comment (secch97) [reply src/index.js:5]: Thank you! I had to search React docs to solve that!

**Pull request merged**